cmd/history.go

```diff
@@ -72,20 +81,20 @@ func ExecuteHistoryCleanupCommand(args []string) error {
 	}
 	var matchingTags = cleanup.GetMatchingTags(&gitCandidates, &imageStreamTags, matchOption)
 
-	activeImageStreamTags, err := openshift.GetActiveImageStreamTags(namespace, image, imageStreamTags)
+	activeImageStreamTags, err := openshift.GetActiveImageStreamTags(namespace, image, matchingTags)
```

Comment:
Doesn't change that the behavior slightly? At least it would only log the images that were matched with the git tag, and not the ones that are non-matching, but still used. could be potentially misleading to the user as it might not print image tags despite still being used?

Reply:
I'd say that's fine especially for the performance boost this brings...